platform_versioned_docs/version-24.3/getting-started/studios.mdx

```diff
@@ -0,0 +1,452 @@
+date: "17 Feb 2025"
+tags: [platform, studios, data studios, jupyter, rstudio, xpra, vscode, conda]
+toc_max_heading_level: 3
+---
+
+[Studios](../data_studios/index.mdx) allows users to host a variety of container images directly in Seqera Platform compute environments for analysis using popular environments including Jupyter (Python) and RStudio notebooks (R), Visual Studio Code IDEs, and Xpra remote desktops. Each studio session provides a dedicated interactive environment that encapsulates the live environment.
+
+This guide explores how Studios integrates with your existing workflows, bridging the gap between pipeline execution and interactive analysis. It details how to set up and use each type of studio, demonstrating a practical use case for each.
+
+:::info[**Prerequisites**]
+You will need the following to get started:
+
+- Valid credentials for your cloud storage account and compute environment
+- To create and configure Studios, you need at least the "Maintain" workspace [user role](../orgs-and-teams/roles.mdx)
+- An AWS Batch workspace compute environment (**without Fargate**) with sufficient resources (minimum: 2 CPUs, 8192 MB memory)
+- [Data Explorer](../data/data-explorer.mdx) enabled in your workspace
+:::
+
+### Jupyter: Python-based visualization of protein structure prediction data 
+
+Studios and Jupyter notebooks enable interactive analysis using Python libraries and tools. For example, PyMOL is a powerful tool used for visualizing and comparing structures produced by workflows such as [nf-core/proteinfold](https://nf-co.re/proteinfold/1.1.1), a bioinformatics best-practice analysis pipeline for protein 3D structure prediction. This section demonstrates how to create an AWS Batch compute environment, add the nf-core AWS megatests public proteinfold data to your workspace, create a Jupyter studio, and run the provided Python script to produce interactive composite 3D images of the H1065 sequence. 
+
+:::note
+This script and instructions can also be used to visualize the structures from nf-core/proteinfold runs performed with your own public or private data. 
+:::
+
+#### Create an AWS Batch compute environment 
+
+Studios requires an AWS Batch compute environment. If you do not have an existing compute environment available, create one with the following attributes:
+
+- **Region**: To minimize costs, your compute environment should be in the same region as your data. To browse the nf-core AWS megatests public data optimally, select **eu-west-1**.
+- **Provisioning model**: Use **On-demand** EC2 instances. 
+- Studios does not support AWS Fargate. Do not enable **Use Fargate for head job**. 
+- At least 2 available CPUs and 8192 MB of RAM. 
+
+#### Add data using Data Explorer
+
+For the purposes of this guide, add the proteinfold results (H1065 sequence) from the nf-core AWS megatests S3 bucket to your workspace using Data Explorer:
```

Comment:
Link to the prefix or is assumption everyone knows what this is?
Nevermind, I see it's 2 lines down.